public/table.js

```diff
@@ -0,0 +1,106 @@
+class TableController {
+    static updateCodingProgressTable(codaData) {
+        let data = codaData.data,
+            lastUpdate = codaData.lastUpdate; 
+
+        // Set last updated timestamp in UI
+        document.getElementById("last-update").innerText = `Last updated: ${lastUpdate}`;
+
+        // Save sorting state
+        let sortInfo = { column: "", order: "" };
+
+        // Invoke `transform` function with column to be sorted on page load
```

Comment:
This works well on page load, but when you sort by a different column e.g NC% and firebase updates the data  (which happens every 10 sec or so) the sorting goes back to default. It would be great if we can maintain user sorting when data updates.

Reply:
@DanielMwendwa  This has not been resolved yet.

Reply:
Have you tested the branch with the changes below?

Reply:
Your right I forgot to pull the new changes. This looks okay now thanks.